content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -200,9 +200,11 @@ and automatically adds a default storage class to them.
 This way, users that do not request any special storage class do not need to care about them at all and they
 will get the default one.
 
-This admission controller does not do anything when no default storage class is configured. When more than one storage
-class is marked as default, it rejects any creation of `PersistentVolumeClaim` with an error and an administrator
-must revisit their `StorageClass` objects and mark only one as default.
+This admission controller does nothing when no default `StorageClass` exists. When more than one storage
+class is marked as default, and you then create a `PersistentVolumeClaim` with no `storageClassName` set, 
+Kubernetes uses the most recently created default `StorageClass`.
+When a `PersistentVolumeClaim` is created with a specified `volumeName`, it remains in a pending state 
+if the static volume's `storageClassName` does not match the `StorageClass` on the `PersistentVolumeClaim`.
```

Comment:
I think the sentence is a bit misleading. If the PVC `storageClassName` is nil, then the static PV and PVC should match *after* a default storage class is set to the PVC. For example:

```suggestion
When a `PersistentVolumeClaim` is created with a specified `volumeName`, it remains in a pending state 
if the static volume's `storageClassName` does not match the `storageClassName` on the `PersistentVolumeClaim` after any default StorageClass is applied to it.
```